docs/mdbook/src/android/getting-started-guide.md

```diff
@@ -0,0 +1,113 @@
+
+    ```gradle
+        dependencies {
+            ...
+            implementation 'org.maplibre.gl:android-sdk:<version>'
+            ...
+        }
+    ```
+
+3. Sync your Android project with Gradle files.
+
+4. Add a `MapView` to your layout XML file (usually `<project>/<app-module>/src/main/res/layout/activity_main.xml`).
+
+    ```xml
+        ...
+        <com.mapbox.mapboxsdk.maps.MapView
+            android:id="@+id/mapView"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            />
+        ...
+    ```
+
+5. Initialize the `MapView` in your `MainActivity` file by following the example below:
+
+    ```kotlin
+
+        ...
+        import androidx.appcompat.app.AppCompatActivity
+        import android.os.Bundle
+        import android.view.LayoutInflater
+        import com.mapbox.mapboxsdk.Mapbox
+        import com.mapbox.mapboxsdk.camera.CameraPosition
+        import com.mapbox.mapboxsdk.geometry.LatLng
+        import com.mapbox.mapboxsdk.maps.MapView
+
+        class MainActivity : AppCompatActivity() {
+
+            // Declare a variable for MapView
+            private var mapView: MapView? = null
+
+            override fun onCreate(savedInstanceState: Bundle?) {
+                super.onCreate(savedInstanceState)
+                
+                // Init MapLibre
+                Mapbox.getInstance(this)
+
+                // Init layout view
+                val inflater = LayoutInflater.from(this)
+                val rootView = inflater.inflate(R.layout.activity_main, null)
+                setContentView(rootView)
+
+                // Init the MapView
+                mapView = rootView.findViewById(R.id.mapView)
+                mapView.getMapAsync { map ->
+                    map.setStyle("https://demotiles.maplibre.org/style.json")
+                    map.cameraPosition = CameraPosition.Builder().target(LatLng(0.0,0.0)).zoom(1.0).build()
+                }
+            }
+
+            override fun onStart() {
+                super.onStart()
+                mapView?.onStart()
+            }
+
+            override fun onResume() {
+                super.onResume()
+                mapView?.onResume()
+            }
+
+            override fun onPause() {
+                super.onPause()
+                mapView?.onPause()
+            }
+
+            override fun onStop() {
+                super.onStop()
+                mapView?.onStop()
+            }
+
+            override fun onLowMemory() {
+                super.onLowMemory()
+                mapView?.onLowMemory()
+            }
+
+            override fun onDestroy() {
+                super.onDestroy()
+                mapView?.onDestroy()
+            }
+        }
+        
```

Comment:
let's also add 

`override fun onSaveInstanceState(outState: Bundle) {
        super.onSaveInstanceState(outState)
        mapView.onSaveInstanceState(outState)
    }`

Reply:
Okay @ovivoda . I added this part to the example in the getting started guide.